src/tests/test_buildercore_terraform.py

```diff
@@ -470,6 +470,46 @@ def test_bigquery_full_template(self):
             'schema': '${file("key-value.json")}',
         })
 
+    def test_bigquery_remote_paths(self):
+        "remote paths require terraform to fetch and load the files, which requires another entry in the 'data' list"
+        pname = "project-with-bigquery-remote-schemas"
+        iid = pname + "--prod"
+        context = cfngen.build_context(pname, stackname=iid)
+        terraform_template = json.loads(terraform.render(context))
+
+        expecting = json.loads('''{
```

Comment:
this is less effective than the standard of this file to use dictionaries; syntax errors in the JSON produce `ValueError`s with the JSON line numbering rather than the `.py` file lines, cannot put comments into it, `"` needs double escaping, etc.

Won't hold this up but needs to be uniformed to the rest